cli/src/main/java/org/apache/uniffle/cli/UniffleCLI.java

```diff
@@ -66,14 +91,40 @@ public int run(String[] args) throws UniffleCliArgsException {
       System.out.println("uniffle-admin-cli : " + cliArgs);
       return 0;
     }
+    if (cmd.hasOption(coordServer.getOpt()) && cmd.hasOption(coordPort.getOpt())) {
+      String host = cmd.getOptionValue(coordServer.getOpt()).trim();
+      int port = Integer.parseInt(cmd.getOptionValue(coordPort.getOpt()).trim());
+      String hostUrl = String.format("http://%s:%d", host, port);
+      LOG.info("uniffle-admin-cli : coordinator server host {}, port {}.", host, port);
```

Comment:
If I understand correctly, the host:port is for all cli options rather than only for admin. So this log is not accurate.

Reply:
How about this?  `connected to coordinator: http://xxxxxx:xxxxxxx`

Reply:
fixed